spec/controllers/teams_controller_spec.rb

```diff
@@ -107,6 +124,14 @@
       }.to change(Team, :count).by(-1)
     end
 
+    it "deletes the associated team ownerships and team memberships" do
+      team = FactoryBot.create(:team)
```

Comment:
Please use consistently either `.create :team` or `.create(:team)`. I'd prefer the option without brackets personally.

Reply:
👍